spring-boot-modules/spring-boot-keycloak/pom.xml

```diff
@@ -114,7 +102,7 @@
     </build>
 
     <properties>
-        <keycloak-adapter-bom.version>15.0.2</keycloak-adapter-bom.version>
+        <keycloak-adapter-bom.version>20.0.2</keycloak-adapter-bom.version>
```

Comment:
can this property be removed?

Reply:
code change done in PR https://github.com/eugenp/tutorials/pull/13293